woeip/templates/registration/login.html

```diff
@@ -1,8 +1,49 @@
 {% extends '__base.html' %}
 {% block content %}
-<form method="post">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <button type="submit">Login</button>
-</form>
+<div class="grid-container large-offset-1">
+  <div class="grid-x">
+    <div class="cell medium-6"><h2>Welcome to WOAQ</h2></div>
+  </div>
+  <div class="grid-x">
+    <div class="cell medium-6">
+      <p>West Oakland Air Quality supports the
+    West Oakland Environmental Indicators Project
+    by putting local air quality data in the hands
+    of West Oaklanders</p>
+    </div>
+  </div>
+    <div class="grid-x">
+      <div class="cell medium-6"><h3>Sign in to upload data</h3></div>
+    </div>
+    <div class="grid-x">
+      <div class="cell medium-6">
+        <p>Or request an account from the WOAQ team at 
+          <a href="mailto:woaq@openoakland.org">woaq@openoakland.org</a>
```

Comment:
The email should probably be passed from the backend as a context variable, but that can happen in the future.